rct229/ruletest_engine/ruletest_jsons/section5/rule_5_49.json

```diff
@@ -0,0 +1,352 @@
+        "Rule": 49,
+        "Test": "a",
+        "test_description": "Building is located in climate zone 4A and includes one conditioned zone which has two surfaces - one adjacent to exterior and another adjacent to ground. The proposed infiltration flow rate at wind pressure is established correctly based on default 0.6 cfm/ft2  @75PA.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "5-49",
+            "ruleset_reference": "Table G3.1 #5 Proposed column (b): The proposed air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 0.6 cfm/ft2 for buildings providing verification in accordance with Section 5.9.1.2. The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4. \nExceptions: When whole-building air leakage testing, in accordance with Section 5.4.3.1.1, is specified during design and completed after construction, the proposed design air leakage rate of the building envelope shall be as measured. ",
+            "rule_description": "The proposed air leakage rate of the building envelope (I75Pa) at a fixed building pressure differential of 0.3 in. of water shall be 0.6 cfm/ft2 for buildings providing verification in accordance with Section 5.9.1.2. The air leakage rate of the building envelope shall be converted to appropriate units for the simulation program using one of the methods in Section G3.1.1.4. \nExceptions: When whole-building air leakage testing, in accordance with Section 5.4.3.1.1, is specified during design and completed after construction, the proposed design air leakage rate of the building envelope shall be as measured.",
+            "applicable_rmr": "Proposed RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "rule_dependency": "None",
+            "primary_rule": "Y",
+            "schema_version": "0.0.12"
+        },
+        "rmr_transformations": {
+            "proposed": {
+                "id": "ASHRAE229",
+                "weather": {
+                    "climate_zone": "CZ4A",
+                    "data_source_type": "OTHER"
+                },
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMR 1",
+                        "measured_infiltration_pressure_difference": 75,
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Zone 1",
```

Comment:
I believe we need ```spaces``` key along with the following keys ```id```, ```occupant_multiplier_schedule```, ```floor_area``` inside the ``` spaces``` key to use the ```get_zone_conditioning_category_dict``` function. I'd appreciate it if you could verify this.